JamesIves/github-pages-deploy-action
# Allow Subsequent Deployments

## Description



Allows subsequent deployments by cleaning up branch artifacts. Also sets up special casing for single commits with a target folder to allow follow-up commits when two deployment jobs are run after another.

## Testing Instructions



* Early preview is available at `JamesIves/github-pages-deploy-action@releases/v4-alpha`. 
* Create a workflow with a deployment job that occurs one after another.
* The job should exit early in most cases except when single-commit or target-folder is configured allowing versioning to properly occur.

## Additional Notes



This PR is realted to #679 

I still need to write unit tests for this and run the integration tests before this can be merged.
